app/components/Explorer/Accounts.jsx

```diff
@@ -73,6 +74,20 @@ class Accounts extends React.Component {
         this.forceUpdate();
     }
 
+    _getBalanceObject(object_id) {
+        let {balanceObjects} = this.state;
+
+        if(object_id && typeof(object_id) === "string") {
+            if(!balanceObjects[object_id]) {
+                balanceObjects[object_id] = parseFloat(ChainStore.getObject(object_id).get("balance"));
+            }
+        }
+
+        this.setState({balanceObjects});
```

Comment:
Is it on purpose that the cache is in the state of this componenet? Will cause heacy re-renderings, won't it?

Reply:
Perhaps, changed to not use setState for the cache.